client/src/components/Fan.tsx

```diff
@@ -95,102 +96,94 @@ const Fan = ( {connectedWallet, provider, onWalletConnectClick, match }:IProps)
     console.log(`Transaction was mined in block ${receipt.blockNumber}`);
     */
-
-  }
-
-  const isValidAddress = (address: string) => {
-    // TODO: Support ENS addresses.
-    return ethers.utils.isAddress(address);
-  }
-
-  const isValidAmount = (amount: number) => {
-    return Number.isFinite(amount) && amount > 0.0;
-  }
-
-    let button;
-
-    if (!connectedWallet) {
-      button = (
-        <Button
-          variant="primary"
-          size="lg"
-          block
-          type="button"
-          onClick={onWalletConnectClick}
-          data-testid="connect-wallet-button"
-        >
-          Connect wallet
-        </Button>
-      );
-    } else {
-      const disabled =
-        !isValidAddress(creatorAddress!) ||
-        !isValidAmount(amountValue!);
-      button = (
-        <Button
-          variant="success"
-          size="lg"
-          block
-          type="submit"
-          disabled={disabled}
-          data-testid="send-button"
-        >
-          Send
-        </Button>
-      );
-    }
-
-    return (
-      <div className="fan-container">
-        <Card className="support-card">
-          <Card.Body>
-            <Card.Title>Support</Card.Title>
-            <Card.Subtitle>Send crypto to your favorite creator.</Card.Subtitle>
-            <Form className="support-form" onSubmit={sendDonation}>
-              <Form.Group controlId="creatorAddress">
-                <Form.Label srOnly>Creator's ETH wallet address</Form.Label>
-                <Form.Control
-                  name="creatorAddress"
-                  placeholder="Creator's ETH address"
-                  type="text"
-                  size="lg"
-                  defaultValue={creatorAddress}
-                  onChange={handleInputChange}
-                  data-testid="creatorAddress"
-                />
-              </Form.Group>
-              <Form.Group controlId="amountValue">
-                <Form.Label srOnly>Amount in ETH</Form.Label>
-                <InputGroup>
-                  <Form.Control
-                    name="amountValue"
-                    size="lg"
-                    type="text"
-                    placeholder="0.0"
-                    onChange={handleInputChange}
-                    data-testid="amountValue"
-                  />
-                  <InputGroup.Append>
-                    <InputGroup.Text>ETH</InputGroup.Text>
-                  </InputGroup.Append>
-                </InputGroup>
-              </Form.Group>
-              {button}
-            </Form>
-          </Card.Body>
-        </Card>
-        <TransactionModal
-          confirmedText="Thank you for supporting the creators you love!"
-          modalState={transactionModalState}
-          transactionHash={sentTransactionHash}
-          onHide={resetTransactionModal}
-        />
-      </div>
-    );
-}
+	};
+
+	const isValidAddress = (address: string) =>
+	// TODO: Support ENS addresses.
```

Comment:
Can we remove this if `ethers` offers out of the box support for ENS addresses?

Reply:
Is Address does not support ENS directly. I will have to use another method, which will be in the next PR.